_includes/footer.html

```diff
@@ -1,2 +1,10 @@
-	</body>
+<aside class="sticky">
+  <p>{{ site.footer_text_pr | markdownify }}</p>
+</aside>
+<footer class="">
+  <p>{{ site.footer_text_instructions | markdownify }}</p>
+  <p class="divider footer">{{ site.contact_email_footer | markdownify }}</p>
```

Comment:
it seems like there's no padding below this, resulting in a very limited touch target. How about 16px, or 24px?